src/components/permissions/routes.ts

```diff
@@ -40,6 +40,7 @@ const routeLogin = async (request: any, reply: any) => {
     if (!(await bcrypt.compare(password, passwordHash as string)))
       return reply.send({ success: false })
 
+    await databaseConnect.updateUserOrgPolicyTemplate(userId, null)
```

Comment:
I have slight concern that we've captured all the events when this needs to be updated.

What if we create a new template version? When would a user's access to that be updated?